src/connection.rs

```diff
@@ -239,6 +239,11 @@ impl Connection {
         unsafe { ffi::sqlite3_total_changes(self.raw.0) as usize }
     }
 
+    #[inline]
+    pub fn last_insert_rowid(&self) -> i64 {
```

Comment:
Please add a docstring, rename to `last_inserted_row`, and changes the output type to `usize`. The documentation says it is signed, but I fail to see how it might return a negative value.